packages/clay-management-toolbar/src/__tests__/index.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayManagementToolbar from '..';
+import React from 'react';
+import {cleanup, render} from '@testing-library/react';
+
+describe('ClayManagementToolbar', () => {
+	afterEach(cleanup);
+
+	it('renders', () => {
```

Comment:
Need we to include more interaction tests? Especially for event callbacks

Reply:
Yeah, there's still a lot to work on testing, I'll add after we are fine with the component approach, I mentioned above in the PR description.